docs/app/pages/components/components-sections/notifications/notifications/notifications.component.html

```diff
@@ -0,0 +1,61 @@
+<ux-notification-list></ux-notification-list>
+
+<button (click)="notificationService.showNotification(myNotification, {})">Show Notification</button>
+<button (click)="notificationService.dismissAllNotifications()">Dismiss Notifications</button>
+
+<hr>
+
+<p>Notifications can be used to inform the user of occurring events or to alert the user of something that may require their
+    attention. Notifications will appear in the top right corner of the browser window and can be dismissed by the user by
+    clicking the close button.</p>
+
+<p>Notifications are controlled by the <code>notificationService</code> which can be injected into any controller in the application.</p>
+
+<p>The content of the notification should be specified inside an <code>ng-template</code> element.</p>
+
+<p>The <code>notificationService</code> has several functions available to control notifications.</p>
+
+<uxd-api-properties>
+    <tr uxd-api-property name="showNotification">
```

Comment:
I know this is assigned to Donna, but I just had a quick read over and have a few minor suggestions.

Consider removing 'notification' from the function names as they are going to be calling `notificationService.xyz()` the notificationService already implies the context is regarding notifications so it can be removed from the function names. Eg:

showNotification -> show
dismissNotification -> dismiss
dismissAllNotifications -> dismissAll
getNotifications -> getHistory? or something similar

Also in the description below:

```
It should be given an ng-template element containing the elements to be displayed and an object containing properties to configure the notification.
```

It should be a `TemplateRef` they are giving to the function rather than the element. You should also make the options parameter optional, I notice in the example above you call:

```
notificationService.showNotification(myNotification, {})
```

Allow them to skip this second argument if they just want to use the default options.